applications/nrf5340_audio/src/modules/Kconfig.defaults

```diff
@@ -45,6 +45,15 @@ config USB_DEVICE_MANUFACTURER
 	string
 	default "Nordic Semiconductor AS"
 
+## Avoid redundant warnings for endpoint setting in USB stack
```

Comment:
Do these need to be part of this PR?

Reply:
One of major purposes of this PR is to elimate kconfig warning messages while compiling.
These 2 kconfigs are only applicable when device is gateway.
They will generate kconfig warnings when compiling headset since headset doesn't enable USB driver.